server/py/framework/utils/clients/iguazio/v4.py

```diff
@@ -26,6 +27,41 @@
 
 
 class Client(BaseClient):
+    def __init__(self, *args, **kwargs) -> None:
+        super().__init__(*args, **kwargs)
+        self._client = iguazio.Client(api_url=self._api_url)
+
+    def refresh_access_token(self, token_name: str, token: str) -> None:
+        """
+        Refresh the access token using the Iguazio client to validate the offline token.
+
+        :param token_name: Logical name of the token (used for error messages)
+        :param token: The offline token string
+        :raises mlrun.errors.MLRunUnauthorizedError: If the token is invalid or expired
+        """
+        try:
+            self._client.refresh_access_token(token)
```

Comment:
is this the new iguazio SDK?
If so, we need to expose this method to the top level client as currently it isn't. Could you please open a ticket for this?